internal/api/connectors/cloudConnector_test.go

```diff
@@ -97,10 +97,11 @@ var _ = Describe("Cloud Connector", func() {
 		err = json.Unmarshal(bytes, &parsedRequest)
 		Expect(err).ToNot(HaveOccurred())
 
-		Expect(parsedRequest["account"]).To(Equal("1234"))
 		Expect(parsedRequest["directive"]).To(Equal("playbook"))
-		Expect(parsedRequest["recipient"]).To(Equal(recipient.String()))
 		Expect(parsedRequest["payload"]).To(Equal(url))
+		Expect(parsedRequest["payload"]).To(ContainSubstring(recipient.String()))
```

Comment:
Nope, we're making an HTTP call to cloud connector. More specifically, to this resource https://github.com/RedHatInsights/cloud-connector/blob/master/internal/controller/api/api.spec.json#L25 and we want to verify what we're properly setting the `client_id` segment (i.e. our recipient id is used there).

Therefore, you want to verify that doer.Request.URL contains the URL.